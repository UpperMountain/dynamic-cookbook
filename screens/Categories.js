import React from "react";
import { Text } from "react-native";
import tabIcon from "../components/tabIcon";
import SafeView from "../components/SafeView";

const Categories = () => (
  <SafeView>
    <Text>Categories</Text>
  </SafeView>
);

Categories.navigationOptions = {
  title: "Categories",
  tabBarIcon: tabIcon("th")
};

export default Categories;
