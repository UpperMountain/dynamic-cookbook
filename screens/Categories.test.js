import React from "react";
import renderer from "react-test-renderer";
import Categories from "./Categories";

it("renders without exploding", () => {
  renderer.create(<Categories />);
});

it("passes snapshot tests", () => {
  const cmp = renderer.create(<Categories />);
  expect(cmp.toJSON()).toMatchSnapshot();
});
