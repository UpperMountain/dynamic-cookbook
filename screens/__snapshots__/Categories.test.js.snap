// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passes snapshot tests 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "flex": 1,
      "margin": 50,
      "paddingTop": 0,
    }
  }
>
  <Text>
    Shadow Sandbox
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 10,
        "elevation": 0,
        "height": 100,
        "margin": 20,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0,
        "shadowRadius": 0,
        "width": 100,
      }
    }
  />
  <Text>
    0
  </Text>
  <RCTSlider
    disabled={false}
    enabled={true}
    max={1}
    maximumValue={1}
    min={0}
    minimumValue={0}
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onValueChange={[Function]}
    step={0}
    style={
      Object {
        "height": 40,
      }
    }
    value={0}
  />
</RCTSafeAreaView>
`;
